google/cloud/aiplatform/lro.py

```diff
@@ -15,6 +15,71 @@
 # limitations under the License.
 #
 
+import proto
+
+from google.api_core import operation as ga_operation
+from google.cloud.aiplatform import base
+from typing import Optional, Callable
+
 
 class LRO:
-    pass
+    """A handler for operation futures."""
+
+    def __init__(
+        self,
+        operation_future: ga_operation.Operation,
+        resource_noun_obj: Optional[base.AiPlatformResourceNoun] = None,
+        result_key: Optional[str] = None,
+        api_get: Optional[Callable[[proto.Message], proto.Message]] = None,
```

Comment:
I think the signature for `api_get` should be `Optional[Callable[..., proto.Message]]` because the GAPIC calls take a variable number of key word arguments.